css-brochure-form/styles.css

```diff
@@ -0,0 +1,64 @@
+h4 {
+  margin-left: 7.6rem;
```

Comment:
This is a lot of margin. If you are using this to center the text, try `text-align: center;`